medusa/medusacli.py

```diff
@@ -47,21 +48,41 @@
 pass_MedusaConfig = click.make_pass_decorator(medusa.config.MedusaConfig)
 
 
-def configure_logging(verbosity, without_log_timestamp):
+def configure_file_logging(config):
```

Comment:
The reason I'm using 2 separate functions is because loading the config _already produces logging output_ so I couldn't find an elegant way of initializing both at the same time and not have the log losing the format.